spec/components/pretty_text_spec.rb

```diff
@@ -798,12 +798,26 @@ def extract_urls(text)
       expect(post.excerpt).to eq("hello <a href=\"https://site.com\" rel=\"noopener nofollow ugc\">site</a>")
     end
 
+    it "handles div excerpt at the beginning of a post" do
+      expect(PrettyText.excerpt("<div class='excerpt'>hi</div> test", 100)).to eq('hi')
+      post = Fabricate(:post, raw: "<div class='excerpt'>hi</div> test")
```

Comment:
Since `post.excerpt` is calling `PrettyText.excerpt`, we don't really need to test both every time? Just test `PrettyText.excerpt` and avoid doing any duplicate work?

Reply:
It looked like a unit test + an integration test on one spec. I removed it from this and several other tests.